lib/app.dart

```diff
@@ -7,6 +9,16 @@ class App extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
       title: F.title,
+      localizationsDelegates: [
+        AppLocalizations.delegate,
+        GlobalMaterialLocalizations.delegate,
+        GlobalWidgetsLocalizations.delegate,
+        GlobalCupertinoLocalizations.delegate
+      ],
+      supportedLocales: [
+        const Locale('en'),
+        const Locale('vi'),
+      ],
```

Comment:
You can also use the generated `localizationsDelegates` and `supportedLocales` list instead of providing them manually.
Refer [this](https://flutter.dev/docs/development/accessibility-and-localization/internationalization)
```suggestion
      localizationsDelegates: AppLocalizations.localizationsDelegates,
      supportedLocales: AppLocalizations.supportedLocales,
```

Reply:
@markgravity thanks, updated 🙏